.github/workflows/rules.yaml

```diff
@@ -0,0 +1,11 @@
+name: Test prometheus rules
```

Comment:
`rules.yaml` doesn't give any context what kind of texts those are. Let's rename the file to `contrib.yaml`

Reply:
Thanks good idea, I changed it.